l10n_nl_tax_statement/views/l10n_nl_vat_statement_view.xml

```diff
@@ -0,0 +1,302 @@
+                            />
+                            <field
+                                name="display_multicompany_fiscal_unit"
+                                invisible="1"
+                                groups="base.group_multi_company"
+                            />
+                            <field
+                                name="multicompany_fiscal_unit"
+                                attrs="{'readonly': [('state','in',['posted','final'])],'invisible': [('display_multicompany_fiscal_unit', '=', False)]}"
+                                groups="base.group_multi_company"
+                            />
+                            <field
+                                name="fiscal_unit_company_ids"
+                                widget="many2many_tags"
+                                domain="[('parent_id', '=', company_id)]"
+                                attrs="{'readonly': [('state','in',['posted','final'])],'invisible': [('multicompany_fiscal_unit', '=', False)]}"
+                                groups="base.group_multi_company"
+                            />
+                            <field
+                                name="date_range_id"
+                                attrs="{'readonly': [('state','in',['posted','final'])]}"
+                            />
+                            <label for="from_date" string="Date" />
+                            <div class="o_row">
+                                <field
+                                    name="from_date"
+                                    widget="daterange"
+                                    nolabel="1"
+                                    class="oe_inline"
+                                    options="{'related_end_date': 'to_date'}"
+                                    attrs="{'readonly': [('state','in',['posted','final'])]}"
+                                />
+                                <i
+                                    class="fa fa-long-arrow-right mx-2"
+                                    aria-label="Arrow icon"
+                                    title="Arrow"
+                                />
+                                <field
+                                    name="to_date"
+                                    widget="daterange"
+                                    nolabel="1"
+                                    class="oe_inline"
+                                    options="{'related_start_date': 'from_date'}"
+                                    attrs="{'readonly': [('state','in',['posted','final'])]}"
+                                />
+                            </div>
+                        </group>
+                        <group name="extra_parameters">
+                            <field name="date_posted" />
+                            <field name="date_update" />
+                            <field name="currency_id" />
+                            <field name="target_move" />
```

Comment:
Might be good to show <field name="is_invoice_basis"/> here?

Reply:
This is a leftover, I'm removing it :+1: